server/middleware/content-negotiation.js

```diff
@@ -1,12 +1,13 @@
 module.exports = (req, res, next)=>{
-	const isImageRequest = req.get('Accept')?.split(',')
-        ?.filter((h)=>!h.includes('q='))
-        ?.every((h)=>/image\/.*/.test(h));
-	if(isImageRequest) {
-		return res.status(406).send({
-			message : 'Request for image at this URL is not supported'
-		});
+	if(! req.url.startsWith('/xssp/')) {
```

Comment:
This line is causing test failures, as it doesn't handle the null case for `req.url`.
It will probably work just fine if it's changed to `req.url?.startsWith(...)`.

Reply:
I remember that fix not working for an inexplicable reason, but it is now.